src/picongpu/submit/submitAction.sh

```diff
@@ -24,6 +24,10 @@ mkdir picongpu
 cp -r $TBG_projectPath/bin picongpu
 cp -r $TBG_projectPath/include picongpu
 cp -r $TBG_projectPath/submit picongpu
+if [ -d $TBG_projectPath/pymodules ]
+then
+  cp -r $TBG_projectPath/pymodules picongpu
```

Comment:
QUESTION:
@ComputationalRadiationPhysics/picongpu-developers is the `picongpu` directory the right place for this python code?